ipywidgets/static/widgets/js/widget.js

```diff
@@ -558,6 +558,22 @@ define(["nbextensions/widgets/widgets/js/utils",
              * Public constructor
              */
             WidgetViewMixin.initialize.apply(this, [parameters]);
+            this.id = utils.uuid();
+            
+            // Create and apply a unique style class name that can be used to
+            // style this view directly.
+            this.styleClassName = 'widget' + '-' + this.model.id + '-' + this.id;
+            this.el.className += ' ' + this.styleClassName;
```

Comment:
Arg @jdfreder this actually broke bqplot. If `el` is an svg element, we cannot set the className.